tests/package/test_models.py

```diff
@@ -40,11 +40,21 @@ def test_save(self):
                                 package="test_package",
                                 body='hi')
             key = metadata.build_s3_key('datapackage.json')
-            metadata.save()
+            metadata.save_metadata()
             obs_list = list(s3.list_objects(Bucket=bucket_name, Prefix=key).\
                             get('Contents'))
             self.assertEqual(1, len(obs_list))
             self.assertEqual(key, obs_list[0]['Key'])
+            res = s3.get_object_acl(Bucket=bucket_name, Key=key)
+
+            public_grants = {
+                'CanonicalUser': 'FULL_CONTROL',
+                'Group': 'READ'
+            }
+
+            for grant in res['Grants']:
```

Comment:
Can you just compare the whole res['Grants'] to the expected value? Currently this test does not assure that all users has read permission and owner has full-control. And the same for all acl tests below.